Library/Homebrew/cmd/info.rb

```diff
@@ -62,16 +62,18 @@ def github_fork
   end
 
   def github_info f
-    if f.tap?
-      user, repo = f.tap.split("/", 2)
```

Comment:
The idea here is to minimize the number of places that have to know how to extract the tap user/repo from the formula path, so I'd prefer not to make this change.